src/responder/secrets/local.c

```diff
@@ -447,6 +468,9 @@ static int local_db_create(TALLOC_CTX *mem_ctx,
     ret = local_db_check_containers(msg, lctx, msg->dn);
     if (ret != EOK) goto done;
 
+    ret = local_db_check_containers_nested_level(lctx, msg->dn);
+    if (ret != EOK) goto done;
```

Comment:
OK, I think we can push your patches as they are (when they pass review) and then add my patches to add more DEBUG messages atop them.